metricbeat/module/vsphere/virtualmachine/virtualmachine.go

```diff
@@ -66,6 +68,27 @@ func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// Get all custom fields.
+	customFieldsOk := true
+
+	fieldMap := make(map[int32]string)
```

Comment:
This name can be misleading, perhaps `customFieldsMap` is more clear? what do you think?